package com;

public class main {

	public static void main(String[] args) {
		
		try {
			int divideByZero = 10 / 0;
			System.out.println("Rest of code in try block");
		}
		
		catch (ArithmeticException e) {
			System.out.println("ArithmeticException =>" + e.getMessage());
		}
		

	}

}



output:
ArithmeticException =>/ by zero